controllers/configurationpolicy_controller.go

```diff
@@ -699,22 +697,24 @@ func (r *ConfigurationPolicyReconciler) handleObjects(
 		// Enforce could clear the objNames array so use name instead
 		relatedObjects = addRelatedObjects(compliant, rsrc, namespace, namespaced, []string{name}, reason)
 	} else {
-		if !exists && objShouldExist {
-			compliant = false
-			rsrcKind = rsrc.Resource
-			reason = reasonWantFoundDNE
-		} else if exists && !objShouldExist {
-			compliant = false
-			rsrcKind = rsrc.Resource
-			reason = reasonWantNotFoundExists
-		} else if !exists && !objShouldExist {
-			compliant = true
-			rsrcKind = rsrc.Resource
-			reason = reasonWantNotFoundDNE
-		} else if exists && objShouldExist {
-			compliant = true
-			rsrcKind = rsrc.Resource
-			reason = reasonWantFoundExists
+		rsrcKind = rsrc.Resource
+
```

Comment:
Go doesn't have the ternary operator and here the previous way looks easier for me, just move out `rsrcKind = rsrc.Resource`. The new way is also fine to me, just personal preference.

I wonder if there is a more elegant syntax sugar to deal with this scenario, maybe K-map but here doesn't worth it.

Reply:
Good point. I'll move it out.